UPGRADE-1.13.md

```diff
@@ -1,5 +1,9 @@
 # UPGRADE FROM `v1.12.X` TO `v1.13.0`
 
+1. Class `Sylius\Component\Core\Promotion\Updater\Rule\TotalOfItemsFromTaxonRuleUpdater` has been deprecated.
+
+1. Class `Sylius\Component\Core\Promotion\Updater\Rule\ContainsProductRuleUpdater` has been deprecated.
```

Comment:
A bit more explanation for those would be nice and also a note that they're no longer used at all since the logic was faulty w/ a link to the issue since it's a bit of a special case